test/browser/lists/menu-item-spec.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {renderIntoDocument} from 'react-addons-test-utils';
+import {expect} from 'chai';
+import MenuItem from 'menus/menu-item';
+
+
+describe('MenuItem', () => {
+  it('should display the parent element as the first element when the DOM is rendered', () => {
+  /***Method 1***/
+    //Set up the page
+    const markup = React.renderToStaticMarkup(
+      <MenuItem key="a" primaryText="parent">
+        <MenuItem key="a1" primaryText="child1" />
+        <MenuItem key="a2" primaryText="child2" />
+      </MenuItem>
+    );
+    //Expected DOM value
+    const expectedMarkUp = '<div><span style="border:10px;background:none;'
```

Comment:
If you can work with the second test '_should display the textContent beginning with parent followed by the child elements_', then you can ignore the test that uses `renderToStaticMarkup`.